activemq-broker/src/main/java/org/apache/activemq/broker/BrokerService.java

```diff
@@ -2735,7 +2743,9 @@ public TransportConnector startTransportConnector(TransportConnector connector)
             connector = registerConnectorMBean(connector);
         }
         connector.getStatistics().setEnabled(enableStatistics);
-        connector.start();
+        if(!connector.isAutoStartDisabled()) {
```

Comment:
```suggestion
        if(connector.isAutoStart()) {
```

As an example of what i mean